rosdep/base.yaml

```
@@ -7160,6 +7160,19 @@ osmium:
   fedora: [libosmium-devel]
   nixos: [libosmium]
   ubuntu: [libosmium-dev]
+paho-mqtt-c:
+  alpine: [paho-mqtt-c]
+  debian:
+    bullseye: [libpaho-mqtt1.3]
```

Since we expect this package to be available in all debian distributions going forward, this can just be:

```suggestion
  debian: [libpaho-mqtt1.3]
```